Automattic/wp-calypso
# Stats: Wire up review button

Related to: https://github.com/Automattic/red-team/issues/154

## Proposed Changes

Wire up the review button so it directs the user the the plugin page on WordPress.org.

## Why are these changes being made?


Part of the User Feedback project.

pejTkB-1Ce-p2

## Testing Instructions



* Launch the Calypso Live site.
* Visit the Traffic page.
* Enable the feature flag: `?flags=stats/user-feedback`
* Click either "Leave a review" button.
* Confirm the reviews page open correctly in another window.
* Confirm the floating panel is no longer visible when returning to the Traffic page.

## Pre-merge Checklist
